tests/php/Security/LogOutAuthenticationHandlerTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace SilverStripe\SessionManager\Tests\Security;
+
+use SilverStripe\Control\Middleware\ConfirmationMiddleware\Url;
+use SilverStripe\Control\Session;
+use SilverStripe\Control\Tests\HttpRequestMockBuilder;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\ORM\FieldType\DBDatetime;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
+use SilverStripe\SessionManager\Control\LoginSessionMiddleware;
+use SilverStripe\SessionManager\Model\LoginSession;
+use SilverStripe\SessionManager\Security\LogOutAuthenticationHandler;
+
+class LogOutAuthenticationHandlerTest extends SapphireTest
+{
+    use HttpRequestMockBuilder;
+
+    protected $usesDatabase = true;
+
+    protected static $fixture_file = 'LogOutAuthenticationHandlerTest.yml';
+
+    public function testLogout()
+    {
+        $session = new Session(['activeLoginSession' => 1]);
```

Comment:
You can retrieve the ID of the fixture programmatically, rather than hardcoding it:

```suggestion
        $sessionID = $this->idFromFixture(LoginSession::class, '1');
        $session = new Session(['activeLoginSession' => $sessionID]);
```